.github/workflows/test-cockroachdb.yaml

```diff
@@ -27,20 +27,9 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-      - name: Start a single CockroachDB instance with Docker
-        env:
-          COCKROACHDB_DOCKER_TAG: 'cockroachdb/cockroach:latest-v21.2'
-        run: |
-          docker pull $COCKROACHDB_DOCKER_TAG
-          docker run -d --name roach --hostname roach -p 26257:26257 -p 8080:8080 -v "${{ github.workspace }}:/app" $COCKROACHDB_DOCKER_TAG start-single-node --insecure
-          sudo apt update && sudo apt install wait-for-it -y
-          wait-for-it -h localhost -p 26257
```

Comment:
I think this part is still needed so a CockroachDB instance is actually started that we can later connect to.

Reply:
Oh I see, interesting. There wasn't a similar line to spin up the SQL server instance on the other workflow, so I removed it. I'll add it back and see how it plays out.

Reply:
SQL Server might already be installed and running on the GitHub Actions container we are using.

Reply:
I believe I've fixed all issues related to the actions and the tests now. The tests run successfully on my machine, and the workflow ran successfully on the [670c541](https://github.com/prisma/prisma-examples/pull/3691/commits/670c54161443557c259548e83cf3b8f92499dec3)